openshift-ci/Dockerfile.tools

```diff
@@ -45,7 +55,7 @@ RUN curl -L -s https://github.com/openshift/origin/releases/download/v3.11.0/ope
 # download, verify and install operator-sdk
 RUN curl -L -s https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu -o operator-sdk \
     && curl -L -s https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu.asc -o operator-sdk.asc \
-    && gpg --keyserver keyserver.ubuntu.com --recv-key 90D2B445 \
+    && gpg --keyserver keyserver.ubuntu.com --recv-key 78086003 \
```

Comment:
just curious, why do we have to do this? In https://github.com/fabric8-services/fabric8-wit/blob/master/Makefile#L269 we had a more straightforward process. Is this one better, maybe? (asking to understand)

Reply:
We don’t have the checksum for operator-sdk. They use asc file and gpg public key for verification instead. See https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md

Reply:
ok, thanks for the clarification, @alexeykazakov